pkg/ddl/ddl_worker.go

```diff
@@ -1382,7 +1378,7 @@ func waitSchemaChanged(d *ddlCtx, waitTime time.Duration, latestSchemaVersion in
 	}()
 
 	if latestSchemaVersion == 0 {
-		logutil.Logger(d.ctx).Info("schema version doesn't change", zap.String("category", "ddl"))
+		logutil.Logger(d.ctx).Info("schema version doesn't change", zap.String("category", "ddl"), zap.Int64("jobID", job.ID))
```

Comment:
Maybe put `zap.String("category", "ddl")` as a context value is better. Thus we don't need to write this everywhere.